--- ./addons/pvr.vdr.vnsi/vdr-plugin-vnsiserver/vnsiclient.c-orig	2014-01-21 04:37:55.000000000 +0100
+++ ./addons/pvr.vdr.vnsi/vdr-plugin-vnsiserver/vnsiclient.c	2014-01-21 04:43:27.305025152 +0100
@@ -1324,7 +1324,7 @@ bool cVNSIClient::processTIMER_Update()
 bool cVNSIClient::processRECORDINGS_GetDiskSpace() /* OPCODE 100 */
 {
   int FreeMB;
-  int Percent = VideoDiskSpace(&FreeMB);
+  int Percent = cVideoDirectory::VideoDiskSpace(&FreeMB);
   int Total   = (FreeMB / (100 - Percent)) * 100;
 
   m_resp->add_U32(Total);
