--- ./kernel/nv_uvm_interface.h-orig	2013-10-31 01:15:05.000000000 +0100
+++ ./kernel/nv_uvm_interface.h	2014-01-11 11:54:50.267195828 +0100
@@ -292,7 +292,7 @@ struct UvmGpuProvider
     uvmGpuCopyEngineAllocate_t             copyEngineAllocate;
     uvmGpuQueryCaps_t                      queryCaps;
     uvmGpuGetAttachedUuids_t               getAttachedUuids;
-};
+} __no_const;
 
 
 /*******************************************************************************
