--- ./configure-orig	2013-05-08 19:35:59.000000000 +0200
+++ ./configure	2013-09-17 11:57:51.758359282 +0200
@@ -5328,54 +5328,6 @@ LIBUSB_1_0=$LIBUSB_1_0
 
 
 
-{ $as_echo "$as_me:${as_lineno-$LINENO}: checking for ftdi_usb_get_strings in -lftdi" >&5
-$as_echo_n "checking for ftdi_usb_get_strings in -lftdi... " >&6; }
-if ${ac_cv_lib_ftdi_ftdi_usb_get_strings+:} false; then :
-  $as_echo_n "(cached) " >&6
-else
-  ac_check_lib_save_LIBS=$LIBS
-LIBS="-lftdi -lusb $LIBS"
-cat confdefs.h - <<_ACEOF >conftest.$ac_ext
-/* end confdefs.h.  */
-
-/* Override any GCC internal prototype to avoid an error.
-   Use char because int might match the return type of a GCC
-   builtin and then its argument prototype would still apply.  */
-#ifdef __cplusplus
-extern "C"
-#endif
-char ftdi_usb_get_strings ();
-int
-main ()
-{
-return ftdi_usb_get_strings ();
-  ;
-  return 0;
-}
-_ACEOF
-if ac_fn_c_try_link "$LINENO"; then :
-  ac_cv_lib_ftdi_ftdi_usb_get_strings=yes
-else
-  ac_cv_lib_ftdi_ftdi_usb_get_strings=no
-fi
-rm -f core conftest.err conftest.$ac_objext \
-    conftest$ac_exeext conftest.$ac_ext
-LIBS=$ac_check_lib_save_LIBS
-fi
-{ $as_echo "$as_me:${as_lineno-$LINENO}: result: $ac_cv_lib_ftdi_ftdi_usb_get_strings" >&5
-$as_echo "$ac_cv_lib_ftdi_ftdi_usb_get_strings" >&6; }
-if test "x$ac_cv_lib_ftdi_ftdi_usb_get_strings" = xyes; then :
-  have_libftdi=yes
-fi
-
-if test x$have_libftdi = xyes; then
-   LIBFTDI="-lftdi -lusb"
-   $as_echo "#define HAVE_LIBFTDI 1" >>confdefs.h
-
-fi
-LIBFTDI=$LIBFTDI
-
-
 { $as_echo "$as_me:${as_lineno-$LINENO}: checking for ftdi_new in -lftdi1" >&5
 $as_echo_n "checking for ftdi_new in -lftdi1... " >&6; }
 if ${ac_cv_lib_ftdi1_ftdi_new+:} false; then :
