--- ./src/dhcp-manager/nm-dhcp-dhcpcd.c-orig	2014-04-24 17:18:15.000000000 +0200
+++ ./src/dhcp-manager/nm-dhcp-dhcpcd.c	2014-07-30 16:08:02.580204293 +0200
@@ -144,14 +144,6 @@ ip4_start (NMDHCPClient *client,
 	g_ptr_array_add (argv, (gpointer) "-c");	/* Set script file */
 	g_ptr_array_add (argv, (gpointer) ACTION_SCRIPT_PATH );
 
-#ifdef DHCPCD_SUPPORTS_IPV6
-	/* IPv4-only for now.  NetworkManager knows better than dhcpcd when to
-	 * run IPv6, and dhcpcd's automatic Router Solicitations cause problems
-	 * with devices that don't expect them.
-	 */
-	g_ptr_array_add (argv, (gpointer) "-4");
-#endif
-
 	if (hostname && strlen (hostname)) {
 		g_ptr_array_add (argv, (gpointer) "-h");	/* Send hostname to DHCP server */
 		g_ptr_array_add (argv, (gpointer) hostname );
