--- ./ConfigDefault.pm-orig	2013-04-19 05:23:23.000000000 +0200
+++ ./ConfigDefault.pm	2013-04-22 13:50:47.421349442 +0200
@@ -172,6 +172,9 @@ sub _options_block {
 # Fortran http://en.wikipedia.org/wiki/Fortran
 --type-add=fortran:ext:f,f77,f90,f95,f03,for,ftn,fpp
 
+# Gentoo ebuild, eclass
+--type-add=gentoo:ext:ebuild,eclass
+
 # Google Go http://golang.org/
 --type-add=go:ext:go
 
--- ./t/ack-filetypes.t-orig	2013-04-17 20:07:25.000000000 +0200
+++ ./t/ack-filetypes.t	2013-04-22 13:49:29.114348078 +0200
@@ -25,6 +25,7 @@ delphi
 elisp
 erlang
 fortran
+gentoo
 go
 groovy
 haskell
