--- ./Makefile-orig	2009-04-25 15:00:25.000000000 +0200
+++ ./Makefile	2013-12-27 01:49:37.391433592 +0100
@@ -35,7 +35,7 @@ LDLIBS += -lpcap -lpthread #-lefence
 # Optional C compiler and linker flags. Typical driftnet builds have support
 # for displaying captured images in an X window, and need the following flags:
 CFLAGS  += `pkg-config --cflags gtk+-2.0` `pkg-config --cflags libpng`
-LDLIBS  += -ljpeg -lgif `pkg-config --libs gtk+-2.0` `pkg-config --cflags libpng`
+LDLIBS  += -ljpeg -lgif `pkg-config --libs gtk+-2.0` `pkg-config --libs libpng`
 
 # Alternatively, you can build a version of driftnet which can only be used
 # in `adjunct' mode as the back end for some other image-processing program. To
