--- ./system/dema/session_decoding.c-orig	2012-09-30 13:57:23.000000000 +0200
+++ ./system/dema/session_decoding.c	2013-05-17 12:59:20.219648013 +0200
@@ -111,7 +111,7 @@ static unsigned short SeDePort(unsigned
         return 0;
     }
 #ifdef SO_REUSEPORT
-    if (setsockopt(sock, SOL_SOCKET, SO_REUSEPORT,
+    if (setsockopt(sd, SOL_SOCKET, SO_REUSEPORT,
                    (char *) &yes, sizeof(yes)) < 0) {
         perror("SO_REUSEPORT");
         close(sd);
