--- ./src/lib/synergy/CMakeLists.txt-orig	2014-09-30 12:32:26.000000000 +0200
+++ ./src/lib/synergy/CMakeLists.txt	2014-09-30 12:37:23.624986765 +0200
@@ -136,5 +136,5 @@ include_directories(${inc})
 add_library(synergy STATIC ${src})
 
 if (UNIX)
-	target_link_libraries(synergy arch client ipc net base platform mt server cryptopp)
+	target_link_libraries(synergy arch client ipc net base platform mt server cryptopp pthread)
 endif()
