--- ./src/synapticsstr.h-orig	2013-12-05 00:45:04.000000000 +0100
+++ ./src/synapticsstr.h	2014-01-14 00:30:11.878301722 +0100
@@ -53,7 +53,7 @@
  *****************************************************************************/
 #define SYNAPTICS_MOVE_HISTORY	5
 #define SYNAPTICS_MAX_TOUCHES	10
-#define SYN_MAX_BUTTONS 12      /* Max number of mouse buttons */
+#define SYN_MAX_BUTTONS 15      /* Max number of mouse buttons */
 
 enum OffState {
     TOUCHPAD_ON = 0,
