--- ./simpleclient/wx/wx.cc-orig	2013-05-18 12:53:37.000000000 +0200
+++ ./simpleclient/wx/wx.cc	2013-05-18 13:26:11.952180689 +0200
@@ -607,7 +607,7 @@ void IAXFrame::OnRegisterMenu(wxCommandE
 	wxTextEntryDialog dialog(this,
 	    wxString("Register with a remote asterisk server",wxConvUTF8),
 	    wxString("Format is user:password@hostname",wxConvUTF8),
-	    wxString(wxGetApp().optRegistration,wxConvUTF8),
+	    wxString(wxGetApp().optRegistration.mb_str(),wxConvUTF8),
 	    wxOK | wxCANCEL);
 
 	if(dialog.ShowModal() == wxID_OK)
@@ -813,7 +813,7 @@ bool IAXClient::OnInit()
 	}
     
 	if(!optDestination.IsEmpty()) 
-	    iaxc_call((char *)optDestination.c_str());
+	    iaxc_call((const char *)optDestination.c_str());
     
 
 	return true; 
