--- ./lib/codec_ffmpeg.c-orig	2013-05-18 12:49:47.000000000 +0200
+++ ./lib/codec_ffmpeg.c	2013-05-18 12:51:48.306144752 +0200
@@ -576,7 +576,6 @@ struct iaxc_video_codec *codec_video_ffm
 			e->avctx->rtp_payload_size = fragsize;
 			e->avctx->flags |=
 				CODEC_FLAG_TRUNCATED |
-				CODEC_FLAG_H263P_SLICE_STRUCT |
 				CODEC_FLAG2_STRICT_GOP |
 				CODEC_FLAG2_LOCAL_HEADER;
 			e->avctx->rtp_callback = encode_rtp_callback;
@@ -591,7 +590,6 @@ struct iaxc_video_codec *codec_video_ffm
 		e->avctx->rtp_callback = encode_rtp_callback;
 		e->avctx->flags |=
 			CODEC_FLAG_TRUNCATED |
-			CODEC_FLAG_H263P_SLICE_STRUCT |
 			CODEC_FLAG2_STRICT_GOP |
 			CODEC_FLAG2_LOCAL_HEADER;
 
@@ -619,9 +617,6 @@ struct iaxc_video_codec *codec_video_ffm
 
 		/* e->avctx->flags2 |= CODEC_FLAG2_8X8DCT; */
 
-		/* Access Unit Delimiters */
-		e->avctx->flags2 |= CODEC_FLAG2_AUD;
-
 		/* Allow b-frames to be used as reference */
 		/* e->avctx->flags2 |= CODEC_FLAG2_BPYRAMID; */
 
